app/assets/javascripts/discourse/tests/acceptance/group-manage-email-settings-test.js

```diff
@@ -341,7 +341,7 @@ acceptance(
 
 //     needs.pretender((server, helper) => {
 //       server.post("/groups/47/test_email_settings", () => {
-//         return helper.response(400, {
```

Comment:
Mostly I like this commit very much, but I'm curious why you updated code in comments? Seems peculiar.

Reply:
The second one will be useful when I uncomment that test. This one, I'm reverting (though will probably change in that separate, second PR 🙂)